builders/tools/explorers/verify-contract.md

```diff
@@ -0,0 +1,102 @@
+
+This page will outline the steps for verifying smart contracts on Moonbeam networks.
+
+## Deploying the Contract {: #deploying-the-contract }
+
+In order to verify a smart contract on a block explorer, the contract must be first deployed on the target network first. For this tutorial, we will be deploying the smart contract to [Moonbase Alpha](/builders/get-started/moonbase/). 
+
+You can check this page for a tutorial on [deploying smart contracts](/builders/interact/eth-libraries/deploy-contract/) using Ethereum libraries on Moonbeam. You may also use a developer tool such as [Remix](/builders/interact/remix/#deploying-a-contract-to-moonbeam-using-remix), [Truffle](/builders/interact/truffle/#deploying-a-contract-to-moonbeam-using-truffle), or another tool if preferred, to deploy the smart contract to Moonbeam. 
+
+We will use the same contract as the above deployment tutorial for our contract verification example. 
+
+The contract used is a simple incrementer, arbitrarily named _Incrementer.sol_, which you can find [here](/snippets/code/web3-contract-local/Incrementer.sol). The Solidity code is the following:
+
+```solidity
+--8<-- 'code/web3-contract-local/Incrementer.sol'
+```
+
+### Collecting Information for Contract Verification
+
+You will need to collect some information related to the contract's compiler and deployment in order to verify it successfully. 
+
+1. Take note of the Solidity compiler version used to compile and deploy the contract. The Solidity compiler version can usually be selected or specified in the deployment tool used
+2. Take note of any SPDX license identifier used at the beginning of the Solidity source file (the example uses MIT license):
+    ```
+    // SPDX-License-Identifier: MIT
+    ```
+3. (Optional) If optimization is enabled during compilation, take note of the value of the optimization runs parameter
+4. (Optional) If the contract constructor method accepts arguments, take note of the [ABI-encoded form](https://docs.soliditylang.org/en/develop/abi-spec.html) of the constructor arguments
+5. After deployment, take note the deployed contract address of the smart contract. The deployment address of the contract can be found either in the console output if using a command line based tool such as Truffle, Hardhat, or an Ethereum library, or it can be copied from the GUI in tools such as Remix IDE
+
+![Example Compiler Options in Remix IDE](/images/builders/tools/explorers/verify-contract/verify-contract-1.png)
+
+## Verify the Contract  {: #verifying-the-contract }
+
+The next step will be verifying the smart contract in an EVM compatible explorer for the Moonbeam network that you deployed to. 
+
+### Verify on Moonscan {: #verify-on-moonscan }
```

Comment:
I would change this to "### Moonscan"  as the word "Verify" is showing already above